app.py

```diff
@@ -9,8 +9,12 @@
 )
 
 
-docx_file = st.file_uploader("Load Sample File",type=['txt'])#,'docx','pdf'])
-# if docx_file:
+docx_file = st.file_uploader("Load Sample File",type=['txt','csv'])#,'docx','pdf'])
+#TODO: add functionality for file upload
+if docx_file:
+	st.write("This functionality is under development")
```

Comment:
okay got it

Reply:
Yeah will these functionalities in the next commit